### ISSUE TITLE
Warn about AutoScaleBaseSize issues with WinForms

### ISSUE THREAD

**Issue by NXTwoThou** [dotnet/upgrade-assistant#547]
There's an issue with WinForm's and AutoScaleBaseSize. It would be nice if the upgrade-assistant made note of it or commented out the value in any designer code to make the upgrade more seemless.

https://github.com/dotnet/winforms/issues/1948 mentions how .NET v.5 forms end up much larger than their v4.8 counterparts with no code changes.

https://developercommunity.visualstudio.com/t/winform-designernet-5-destroys-size-values-of-cont/1431491 is a WinForm designer bug I ran into where if AutoScaleBaseSize was set, control size/positions get destroyed on first load.

Since AutoScaleBaseSize was set by the old WinForms designer, it's likely an issue with every project upgraded.

**Comment by docfresh** [comment]
Makes sense to me.  Changing to AutoScaleDimensions prevented our forms from being warped.  But it had to be set before the designer opened.  We had to go back and redo a few forms.    Since changing AutoScaleBaseSize to AutoScaleDimensions appears to be a solid fix, it makes sense the assistant should do it automatically, or at least warn upon finding AutoScaleBaseSize.
